engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/SchemaLogEntryDto.ftl

```diff
@@ -0,0 +1,22 @@
+{
+  "type" : "object",
+  "properties" : {
+
+    <@lib.property
+        name = "id"
+        type = "string"
+        desc = "The id of the schema log entry." />
+
+    <@lib.property
+        name = "timestamp"
+        type = "string"
```

Comment:
we should put `format` for the dates, please check the [readme](https://github.com/camunda/camunda-bpm-platform/tree/master/engine-rest/engine-rest-openapi#formats)
```suggestion
        type = "string"
        format = "date-time"
```

Reply:
Thanks! I'll squash everything with the endpoint path change.